src/SignalR/clients/java/signalr/messagepack/build.gradle

```diff
@@ -0,0 +1,50 @@
+group 'com.microsoft.signalr.messagepack'
+
+dependencies {
+    implementation project(':core')
+    compile 'org.msgpack:msgpack-core:0.8.20'
+    compile 'org.msgpack:jackson-dataformat-msgpack:0.8.20'
```

Comment:
When I try using `compile` in build.gradle it warns "`compile` is deprecated; replace with `implementation`"

Reply:
I saw this in Android Studio

![image](https://user-images.githubusercontent.com/54385/91898868-3d1c6a80-ec51-11ea-9b44-53c7ea7e3f49.png)